test/system/deposit_item_test.rb

```diff
@@ -84,6 +84,11 @@ def before_all
     assert_text I18n.t('items.draft.successful_deposit')
     assert_selector 'h1', text: Item.last.title
 
+    # Check to make sure there isn't any embargo_history
+    item_id = current_url.split('/')[-1]
```

Comment:
I wasn't sure what the [-1] was doing here until I looked it up.  Could do 
```suggestion
    item_id = current_url.split('/').last
```

Reply:
@ConnorSheremeta do you have any thoughts about this?

Reply:
Using .last is definitely more clear in what it's doing, committing your suggestion